cluster/src/main/java/org/apache/iotdb/cluster/utils/hash/Router.java

```diff
@@ -19,8 +19,10 @@
 package org.apache.iotdb.cluster.utils.hash;
 
 import com.alipay.sofa.jraft.util.OnlyForTest;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
```

Comment:
why ArrayList and List are imported here?

Reply:
Removed.